adlfs/core.py

```diff
@@ -170,12 +171,14 @@ def __setstate__(self, state):
 
 
 class AzureDatalakeFile(AzureDLFile):
-    def __init__(self, fs, path, mode="rb", blocksize=2 ** 25, delimiter=None):
+    # TODO: refoctor this. I suspect we actually want to compose an
+    # AbstractBufferedFile with an AzureDLFile.
+    def __init__(self, fs, path, mode="rb", block_size=2 ** 25, delimiter=None):
         super().__init__(
             azure=fs.azure_fs,
             path=AzureDLPath(path),
             mode=mode,
-            blocksize=blocksize,
+            blocksize=block_size,
             delimiter=delimiter,
```

Comment:
was cache_type leaking to here in latest fsspec?

Reply:
Fixed, maybe. Need to actually test it though.